index.html

```diff
@@ -9,18 +9,46 @@
   <script src="scriptForClasses.js" defer></script>
 </head>
 <body>
+  <header>
+    <h2>Awesome books</h2>
+    <nav>
+      <div class="list-tab red">List</div>
+      <div>|</div>
+      <div class="form-tab">Add new</div>
+      <div>|</div>
+      <div class="contact-tab">Contact</div>
+    </nav>
+  </header>
```

Comment:
- [ ] Good job on implementing all the requirements. Everything looks perfect :clap: . Except one, you have forgotten to add the time function to display the current time and date under the navigation bar. look to the design :point_down: 

![778888](https://user-images.githubusercontent.com/57562869/148292080-5a09061e-2f7c-47eb-9433-528d581cc2d0.png)

Reply:
<img src="https://media2.giphy.com/media/xT5LMESsx1kUe8Hiyk/giphy.gif"/>

Reply:
# Solved
<img width="375" alt="Captura de Pantalla 2022-01-05 a la(s) 16 51 00" src="https://user-images.githubusercontent.com/36294012/148300956-e2056e8e-bbf1-41c5-9821-e1adc084fd7d.png">